src/hooks/useSettings.ts

```diff
@@ -0,0 +1,28 @@
+import { useMemo } from "react";
+
+import { DEFAULT_SHOW_CHAT } from "settings";
+
+import { Settings } from "types/Firestore";
+import { SparkleSelector } from "types/SparkleSelector";
+
+import { useFirestoreConnect } from "hooks/useFirestoreConnect";
+import { useSelector } from "hooks/useSelector";
+
+export const settingsSelector: SparkleSelector<Settings | undefined> = (
+  state
+) => state.firestore.data.settings;
```

Comment:
Move to `selectors.ts` as an ordered/hashed pair ?

Example:
```ts
export const ownedVenuesDataSelector: SparkleSelector<
  Record<string, AnyVenue> | undefined
> = (state) => state.firestore.data.ownedVenues;

export const ownedVenuesSelector: SparkleSelector<
  WithId<AnyVenue>[] | undefined
> = (state) => state.firestore.ordered.ownedVenues;
```

Reply:
Hm... looking at this, might not be feasible to have an ordered version I guess

Reply:
hashed version is enough. Move to selectors

Reply:
why do we need ordered here? 😬

Reply:
Not saying we do, ( if you meant https://github.com/sparkletown/sparkle/pull/2261#discussion_r702970543 ). I just copied both as to keep the naming with the `Data` segment as unordered

Reply:
moved here: https://github.com/sparkletown/sparkle/pull/2261/commits/481c1df6d605841a96963ce3f6e73c9f01961154